### PR TITLE
Add release notes for v22.2.19

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-9749

### CODE REVIEW

File: src/current/_includes/releases/v22.2/v22.2.19.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## v22.2.19
+
+Release Date: February 26, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-19-bug-fixes">Bug fixes</h3>
+
+- Querying the [`crdb_internal.leases` table](https://www.cockroachlabs.com/docs/v22.2/crdb-internal) could cause a [node](https://www.cockroachlabs.com/docs/v22.2/ui-cluster-overview-page#node-status) to become unavailable due to a deadlock in the leasing subsystem. [#119463][#119463]
```
Comment (taroface) [inline src/current/_includes/releases/v22.2/v22.2.19.md:9]:
```suggestion
- Previously, querying the [`crdb_internal.leases` table](https://www.cockroachlabs.com/docs/v22.2/crdb-internal) could cause a [node](https://www.cockroachlabs.com/docs/v22.2/ui-cluster-overview-page#node-status) to become unavailable due to a deadlock in the leasing subsystem. This is now fixed. [#119463][#119463]
```

**Review by taroface — APPROVED** [review]
One suggestion but LGTM

File: src/current/_includes/releases/v22.2/v22.2.19.md (hunk 1 above)
Comment (rmloveland) [reply src/current/_includes/releases/v22.2/v22.2.19.md]: Updated - thanks!

**Comment by rmloveland** [comment]
hi @taroface mind giving this another review?  has updated SHA and 1 additional backported PR

v22.2.19 is scheduled for release Monday 2024-02-26

**Comment by taroface** [comment]
LGTM still!

**Pull request merged**